### ISSUE TITLE
ACP template should better reflect actual feedback latency

### ISSUE THREAD

**Issue by scottmcm** [rust-lang/libs-team#180]
The ACP template currently says

https://github.com/rust-lang/libs-team/blob/397fd6aeb9754a12bf5915da5da41f7af115208e/.github/ISSUE_TEMPLATE/api-change-proposal.md#L29

However, near as I can tell, ACPs are *not* reviewed in the weekly meetings -- at least there's no notes about them in minutes like <https://hackmd.io/l8zlGD8cQqeYSfHKG3O33A>.

And thus it seems like setting the expectation of "a week or two" for feedback is misleading.  It's easy to find examples without feedback for months (#115 #67 #63).

**Comment by pitaj** [comment]
I realize the team is busy and stretched thin, but [the stats do not look good](https://pitaj.github.io/rustc-pr-tracking/libs-team.html). ACPs are sitting without activity for many months, even after they enter Initial Comment Period.

**Comment by thomcc** [comment]
I wonder if given the situation it's worth re-evaluating the ACP process, and adjusting or removing it. I feel like the status quo might be worse than what we had before...

**Comment by pitaj** [comment]
Yeah I can personally say it's very disheartening to create an ACP and get zero feedback, even after many months 😞

**Comment by BurntSushi** [comment]
Okay, I merged #219 to at least update the text to give a more accurate expectation.

I do somewhat feel like promising a two week response time is a bit of a high bar to meet.

Another thing to consider is that we, as a team, are overall pretty conservative with what we let into std. Primarily because its support window is "approximately forever." So we should in general expect this sort of process to go somewhat slowly. Whether that also means we have to also respond to literally every incoming request within a short time window is perhaps a different problem though.

**Comment by ChrisDenton** [comment]
I think it would be nice for a welcoming committee (not necessarily libs-api themselves) to give a non-official thanks and perhaps suggestions (e.g. if the proposal needs more motivation or its explanation is lacking in some way) within a few weeks if possible? Both to reassure the contributor they've been heard and to reduce the time libs-api ultimately need to spend on ACPs.

Some kind of triage would probably be good here too.

**Comment by pitaj** [comment]
Can we nominate this for discussion in the next libs-api meeting?

**Comment by Amanieu** [comment]
We will be having an in-person library team meetup in a week, and this is one of the topics that we will want to spend a significant amount of time on.

**Comment by pitaj** [comment]
New process is proposed here

https://github.com/rust-lang/std-dev-guide/pull/55
